src/_posts/2025-12-12-case-study-cdt.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: blog_post
+title: "Case Study: Strengthening California’s Digital Disaster Recovery"
+subtitle: "Supporting the California Department of Technology During the 2025 Wildfire Emergency"
+description: "Supporting the California Department of Technology During the 2025 Wildfire Emergency"
+author: Milo Green
+excerpt: "Supporting the California Department of Technology During the 2025 Wildfire Emergency"
+date: 2025-12-10 00:00:00 +0000
```

Comment:
I will change this date to whatever it is right before we post. 

Currently it's set to a past date so that we can see the post in the preview site.

Reply:
```suggestion
date: 2025-12-15 00:00:00 +0000
```